yaml/parse.ts

```diff
@@ -36,6 +37,23 @@ export interface ParseOptions {
  *
  * Returns a JavaScript object or throws `YAMLError` on error.
```

Comment:
Are you able to replace this line with a [`@throws`](https://jsdoc.app/tags-throws) tag?